packages/ensnode-utils/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "bundler",
+    "module": "ESNext",
+    "noEmit": true,
+    "lib": ["ES2022"],
+    "target": "ES2022",
+    "declaration": true,
+    "outDir": "dist",
+    "strict": true,
+    "allowImportingTsExtensions": true,
+    "emitDeclarationOnly": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true
+  },
+  "include": [
+    "src/**/*",
+    "../../apps/ensnode/src/lib/plugin-helpers.ts",
+    "../../apps/ensnode/test/plugin-helpers.spec.ts"
+  ],
+  "exclude": ["*spec.ts"]
```

Comment:
necessary?

Reply:
Not really if we don't plan to emit types and share them as an npm package.